.github/workflows/node-zxc-build-release-artifact.yaml

```diff
@@ -347,6 +360,14 @@ jobs:
           destination: ${{ secrets.bucket-name }}
           parent: false
 
+      - name: Upload Artifacts (CDN Bucket)
+        uses: google-github-actions/upload-cloud-storage@e95a15f226403ed658d3e65f40205649f342ba2c # v1.0.3
+        if: ${{ inputs.dry-run-enabled != true && needs.validate.outputs.prerelease != 'true' && !cancelled() && !failure() }}
+        with:
+          path: ${{ steps.artifact-release.outputs.folder }}
+          destination: ${{ secrets.cdn-bucket-name }}/node/software/v${{ needs.validate.outputs.version-prefix }}/
```

Comment:
Could you double check if is is '..version-prefix' or just 'prefix' as output earlier.

Reply:
This one is correct because it comes from the job level variable which is `version-prefix`